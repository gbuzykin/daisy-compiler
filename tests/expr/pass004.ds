namespace std {

func f (x : int, y : int) -> int {
  let m = while x < y {}; m
}

}
