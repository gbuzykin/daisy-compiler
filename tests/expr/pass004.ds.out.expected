./expr/pass004.ds:4:7: debug: defining variable `m`
 4 |   let m = while x < y {}; m
   |       ^
./expr/pass004.ds:3:1: debug: function `func f(i32, i32) -> i32` definition
 3 | func f(x : int, y : int) -> int {
   | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./expr/pass004.ds: info: warnings 0, errors 0
